services/src/contexts/migrations/current_schema.sql

```diff
@@ -0,0 +1,875 @@
+CREATE TABLE geoengine (
+    clear_database_on_start boolean NOT NULL DEFAULT false,
+    database_version text NOT NULL
+);
+
```

Comment:
do you want to use the current_schema for uninitialized databases or is this a separate task?

Reply:
Yes, separate task. First merge this one.